DeviceProfileSample/README.md

```diff
@@ -1,25 +1,27 @@
 # Device Profile Sample
 
-For a headless text output client application, it is not possible authenticate through an interactive prompt. Instead a text only approach is necessary. This flow leverages a user's external device (i.e. phone) to authenticate through an interactive login prompt and pass the auth token to the headless application. For more information [click here](https://azure.microsoft.com/en-us/resources/samples/active-directory-dotnet-deviceprofile/?v=17.23h).
+For a headless text output client application, it is not possible authenticate through an interactive prompt. Instead a text only approach is necessary. This flow leverages a user's external device (i.e. phone) to authenticate through an interactive login prompt and pass the auth token to the headless application. For more information [click here](https://azure.microsoft.com/resources/samples/active-directory-dotnet-deviceprofile/?v=17.23h).
```

Comment:
Maybe say that if the tenant admin requires device authentication conditional access policies, using the Device profile flow won't be a good option ?

Reply:
Added the info in Readme.
Thank you.